tensorflow/examples/udacity/1_notmnist.ipynb

```diff
@@ -191,7 +191,8 @@
         "  tar.extractall()\n",
         "  tar.close()\n",
         "  data_folders = [\n",
-        "    os.path.join(root, d) for d in sorted(os.listdir(root)) if d != '.DS_Store']\n",
+        "    os.path.join(root, d) for d in sorted(os.listdir(root))\n",
```

Comment:
A similar change should be pushed to github soon. (I couldn't take in PRs for the last few days due to sync issues)